.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+# On every pull request, but only on push to master
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install affine
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .["test"]
+
+      - name: Run lint and test
+        run: |
+          flake8
+          python -m pydocstyle affine
+          python -m pytest --cov affine --cov-report term-missing
+
+      - name: Coveralls
```

Comment:
We only need to do coverage on one Python version. The code doesn't vary by version.

Reply:
Likewise for flake8 and pydocstyle?

Reply:
well it's doesn't really matter or cost anything to run it on every matrix IMO 🤷

Reply:
Think of the polar bears!